### DISCUSSION TITLE
Create a Reader like Adobe has for Acrobat please

### DISCUSSION THREAD

**Discussion by Snoopycrab** [jgraph/drawio-desktop#1717] [General]
I would love to see a Windows draw.io Reader program so that non-editing users at my company don't open read only vaulted files into edit mode.  Even an option to open existing program in view only mode would be cool.

**Comment by davidjgraph** [comment]
Why not export as PNG+XML or PDF+XML? If the file permission is read-only, the desktop app shouldn't allow editing.

**Comment by Snoopycrab** [comment]
One file would be best...maybe web browsers should be able to view .drawio xml files in the future for people viewing so they don't have to install any program.  Maybe we'll end up vaulting the files and installing drawio on all computers...we don't care much when people open office files in edit mode in 365.  Not being a programmer I thought it would be easy to create a paired down viewer program.
Thanks for that idea.

**Reply by davidjgraph** [reply]
That is one file, a PNG with the XML embedded, or PDF with the XML embedded.

**Reply by Snoopycrab** [reply]
Sorry... I should have said it would be nice to just have the original file
